content/en/agent/guide/private-link.md

```diff
@@ -27,37 +27,37 @@ The overall process consists of configuring an internal endpoint in your VPC tha
 1. Connect to the AWS console and create a new VPC endpoint:
    {{< img src="agent/guide/private_link/create_vpc_endpoint.png" alt="Create VPC endpoint" style="width:60%;" >}}
 2. Select **Find service by name**.
-3. Fill the _service name_ text box with the following value depending of which private link you want to setup:
+3. Fill the _service name_ text box with the following value, depending on which private link you want to set up:
 
     {{< img src="agent/guide/private_link/vpc_service_name.png" alt="VPC service name" style="width:70%;" >}}
     {{< tabs >}}
 
 {{% tab "Metrics" %}}
 
-| Datadog Metric Service name                                |
+| Datadog Metric Service Name                                |
 | ---------------------------------------------------------- |
 | `com.amazonaws.vpce.us-east-1.vpce-svc-056576c12b36056ca`  |
 
 {{% /tab %}}
 {{% tab "Logs" %}}
 
-| Datadog Log Service name                                  |
+| Datadog Log Service Name                                  |
 | --------------------------------------------------------- |
 | `com.amazonaws.vpce.us-east-1.vpce-svc-0a2aef8496ee043bf` |
 
 {{% /tab %}}
 {{% tab "API" %}}
 
-| Datadog API Service name                                  |
+| Datadog API Service Name                                  |
 | --------------------------------------------------------- |
 | `com.amazonaws.vpce.us-east-1.vpce-svc-02a4a57bc703929a0` |
 
 {{% /tab %}}
 {{< /tabs >}}
 
-4. Hit the _verify_ button. If it does not return _Service name found_, reach out to [Datadog support team][2].
+4. Hit the _verify_ button. If it does not return _Service name found_, reach out to the [Datadog support team][2].
 5. Choose the VPC and subnets that should be peered with the Datadog VPC service endpoint.
-6. Make sure that for **Enable DNS name** the _Enable for this Endpoint_ is checked:
+6. Make sure that for **Enable DNS name** the _Enable for this endpoint_ is checked:
```

Comment:
Endpoint is not capitalized on the AWS page:

![image](https://user-images.githubusercontent.com/5319942/75368986-3bc84a80-5888-11ea-8a5a-91e349c8212d.png)